tardis_kromer_plot.py

```diff
@@ -304,17 +309,77 @@ def line_in_L(self):
     @property
     def line_info(self):
         """produces list of elements to be included in the kromer plot"""
-        line_out_infos_within_xlims = self.line_out_infos.loc[
+        self.line_out_infos_within_xlims = self.line_out_infos.loc[
             (self.line_out_infos.wavelength >= self._xlim[0])
             & (self.line_out_infos.wavelength <= self._xlim[1])
         ]
-        self._elements_in_kromer_plot = np.c_[
-            np.unique(
-                line_out_infos_within_xlims.atomic_number.values,
-                return_counts=True,
+
+        """this generates the 4-digit ID for all transitions in the model (e.g. Fe III line --> 2602)"""
+        self.line_out_infos_within_xlims["ion_id"] = (
```

Comment:
comments should be commented with # instead of """ - I know I started the commenting via this method, but these should be changed. Also, they should be truncated at 80 chars and spread over multiple lines if necessary